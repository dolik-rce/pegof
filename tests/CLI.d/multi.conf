input CLI.d/main.peg
input CLI.d/a.peg
input CLI.d/b.peg
