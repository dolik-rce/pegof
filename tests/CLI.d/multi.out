start <- A B

A <- "A"

B <- "B"
X_A <- "X" "_" "A"
X_B <- "X" "_" "B"
